src/agentscope/memory/temporary_memory.py

```diff
@@ -120,16 +135,16 @@ def export(
 
     def load(
         self,
-        memories: Union[str, dict, list],
+        memories: Union[str, list, MessageBase],
         overwrite: bool = False,
     ) -> None:
         if isinstance(memories, str):
```

Comment:
Please check if this is a bug when loading Tht objects
![image](https://github.com/modelscope/agentscope/assets/102287034/e625fc8e-aeed-4385-a97e-b47009c205d5)